cranelift/codegen/src/isa/x64/lower.rs

```diff
@@ -24,6 +25,31 @@ type Ctx<'a> = &'a mut dyn LowerCtx<I = Inst>;
 //=============================================================================
 // Helpers for instruction lowering.
 
+pub(crate) fn ldst_offset(data: &InstructionData) -> Option<i32> {
+    match data {
+        &InstructionData::Load { offset, .. }
+        | &InstructionData::StackLoad { offset, .. }
+        | &InstructionData::LoadComplex { offset, .. }
+        | &InstructionData::Store { offset, .. }
+        | &InstructionData::StackStore { offset, .. }
+        | &InstructionData::StoreComplex { offset, .. } => Some(offset.into()),
+        _ => None,
+    }
+}
+
+pub(crate) fn ty_is_int(ty: Type) -> bool {
+    match ty {
+        B1 | B8 | I8 | B16 | I16 | B32 | I32 | B64 | I64 => true,
+        F32 | F64 | B128 | I128 | I8X16 => false,
+        IFLAGS | FFLAGS => panic!("Unexpected flags type"),
+        _ => panic!("ty_is_int() on unknown type: {:?}", ty),
+    }
+}
+
+pub(crate) fn ty_is_float(ty: Type) -> bool {
+    !ty_is_int(ty)
```

Comment:
`ty.is_float()`

Reply:
Hi thanks. This was copied over from the Aarch64 for prep for some lowering that is not included in this patch. I will use your suggestion when needed.